mathics/core/load_builtin.py

```diff
@@ -144,6 +145,12 @@ def import_and_load_builtins():
     """
     Imports Builtin modules in mathics.builtin and add rules, and definitions from that.
     """
+    # TODO: Check if this is the expected behavior, or it the structures
```

Comment:
@rocky, please check if this is the expected behavior.

Reply:
I think think this is correct. 

(And if not we can adjust later).